src/api/status/src/js/feed-stats.js

```diff
@@ -3,9 +3,8 @@ const { fetch } = require('@senecacdot/satellite');
 module.exports = async function getFeedCount() {
   let feedCount = 0;
   try {
-    const data = await fetch(`${process.env.API_URL}/v1/posts/feeds`, {
-      method: 'HEAD',
-    });
+    const postsFeedURL = `${process.env.POSTS_URL || `${process.env.API_URL}/posts`}/feeds`;
```

Comment:
Let's not do this.  We should be able to rely on `POSTS_URL` in all cases.  If you need to pass it in for `npm` based startup, put it in `env.local`

Reply:
Fixed